admin-dev/themes/new-theme/js/pages/product/components/combination-modal/CombinationModal.vue

```diff
@@ -158,45 +210,134 @@
       onFrameLoaded() {
         this.loadingCombinationForm = false;
         this.submittingCombinationForm = false;
+        const iframeBody = this.$refs.iframe.contentDocument.body;
         this.applyIframeStyling();
+        this.selectedCombinationName = iframeBody.querySelector(ProductMap.combinations.combinationName).innerHTML;
+
+        const iframeInputs = iframeBody.querySelectorAll(
+          ProductMap.combinations.editionFormInputs,
+        );
+
+        iframeInputs.forEach((input) => {
```

Comment:
We should check for `change` event on select, checboxes and radio buttons as well

Reply:
You're right for selects, but checkboxes and radio are inputs